storage/postgresql/pgbench_test.sh

```diff
@@ -136,29 +136,63 @@ function create_pod {
+            		fi 
+		done
+	done 
 
-        done 
+	if [ -n "$resultdir" ]; then
+		for thread in $(echo ${threads} | sed -e s/,/" "/g); do
+			grep "including" $resultdir/threads_${thread}_pgbench_run.txt | cut -d'=' -f2 |cut -d' ' -f2 >> $resultdir/threads_${thread}_including_connections_establishing.txt
+		done 
+	elif [ ! -z "$benchmark_run_dir" ]; then
+		for thread in $(echo ${threads} | sed -e s/,/" "/g); do
+			grep "including" $benchmark_run_dir/threads_${thread}_pgbench_run.txt | cut -d'=' -f2 |cut -d' ' -f2 >> $benchmark_run_dir/threads_${thread}_including_connections_establishing.txt
+		done 
+	fi 
 } 
 
+### draw results 
+
+function draw_result {
+
+	if [ -n "$resultdir" ] ; then 
+		for thread in $(echo ${threads} | sed -e s/,/" "/g); do
+			echo "Thread-${thread}" > $resultdir/results_thread_${thread}.csv 
+			cat  $resultdir/threads_${thread}_including_connections_establishing.txt >>  $resultdir/results_thread_${thread}.csv
+		done 
+
+		paste -d',' ${resultdir}/results_thread_*  >> ${resultdir}/results_storageclass_${storageclass}_clients_${clients}_transactions_${transactions}_scaling_${scaling}.csv
+		curl -o ${resultdir}/drawresults.py https://raw.githubusercontent.com/ekuric/openshift/master/postgresql/drawresults.py
+
+		python ${resultdir}/drawresults.py -r ${resultdir}/results_storageclass_${storageclass}_clients_${clients}_transactions_${transactions}_scaling_${scaling}.csv -i ff -o ${resultdir}/pgbench_results_storageclass_${storageclass}_clients_${clients}_transactions_${transactions}_scaling_${scaling} -t "Clients:${clients} - Transactions:${transactions} - Scaling:${scaling}" -p bars -x "Test iteration" -y "Transactions/Second (tps/sec)" --series=${iterations}
+
+	elif [ ! -z "$benchmark_run_dir" ] ; then 
+		for thread in $(echo ${threads} | sed -e s/,/" "/g); do
+			echo "Thread-${thread}" > $benchmark_run_dir/results_thread_${thread}.csv
+			cat $benchmark_run_dir/threads_${thread}_including_connections_establishing.txt >>  $benchmark_run_dir/results_thread_${thread}.csv
+		done 
+
+		paste -d',' $benchmark_run_dir/results_thread_*  >> $benchmark_run_dir/results_storageclass_${storageclass}_clients_${clients}_transactions_${transactions}_scaling_${scaling}.csv
+                curl -o $benchmark_run_dir/drawresults.py https://raw.githubusercontent.com/ekuric/openshift/master/postgresql/drawresults.py
+
+		python $benchmark_run_dir/drawresults.py -r $benchmark_run_dir/results_storageclass_${storageclass}_clients_${clients}_transactions_${transactions}_scaling_${scaling}.csv -i ff -o $benchmark_run_dir/pgbench_results_storageclass_${storageclass}_clients_${clients}_transactions_${transactions}_scaling_${scaling} -t "Clients:${clients} - Transactions:${transactions} - Scaling:${scaling}" -p bars -x "Test iteration" -y "Transactions/Second (tps/sec)" --series=${iterations}
```

Comment:
This might be my bias: Please ignore if you do not think so. :)
Not very cool to hide yum/pip command in the python script.